CHANGELOG.md

```diff
@@ -8,6 +8,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - `package-lock.json` for `npm5` users
+- Forced update to the newest resin-sdk (^6.4.1)
+- Allow OS version selection when doing `resin device init`
+- Actually tolerate the `--yes` param to `resin device init`
+- Allows passing `--drive` to `resin device init`
+- List detected drives with `resin os available-drives`
```

Comment:
Ok, we can keep it, but I'd still rather have it outside `os`. If we can't come up with anything better then `resin available-drives` feels better than `resin os available-drives` to me (although I would rather namespace it if possible).